### ISSUE TITLE
Feedback the default tab order issue not fix in sample code | incorrect order: username >> signin button>> password

### ISSUE THREAD

**Issue by TonySh127-ms** [azure-ad-b2c/html-templates#8]
Hi Team,

The expected tab order is username >>password >> sign-in button. But if I use the sample code https://github.com/azure-ad-b2c/html-templates/blob/main/templates/AzureBlue/unified.html, the order will be username >> signin button >> password. It cannot fix by upgrading the api.signin content definition version 2.1.7.

I have test directly download the default blue ocean sign-in  html 2.1.7 version by browser tool. The issue cannot reproduce. I also test directly use default ui definition without custom UI. The issue cannot reproduce either.

It seems to relate the current blue ocean html sample code is not up-to-date. https://github.com/azure-ad-b2c/html-templates/blob/main/templates/AzureBlue/unified.html
 
[Download by browser tool.txt](https://github.com/azure-ad-b2c/html-templates/files/8502780/Download.by.browser.tool.txt)

**Comment by Brucelees** [comment]
Hi I am seeeing tab order like Email>> ForgotPassword>>Password >> login button. 

this is wrong order. 

is there a way to fix this.
